src/SDKs/FrontDoor/FrontDoor.Tests/FrontDoor.Tests.csproj

```diff
@@ -5,7 +5,7 @@
     <Description>FrontDoor.Tests Class Library</Description>
     <AssemblyName>FrontDoor.Tests</AssemblyName>
     <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
-    <Version>1.0.0</Version>    
+    <Version>0.9.0-preview</Version>    
```

Comment:
Test project should be set to `1.0.0`

Reply:
How about the version in assemblyinfo.cs (for test project)?

Reply:
should be `1.0.0.0`

Reply:
version updated ,thanks!